src/v/cluster/topic_table.cc

```diff
@@ -796,6 +801,22 @@ std::error_code topic_table::validate_force_reconfigurable_partitions(
     return result;
 }
 
+bool topic_table::topic_multi_property_validation(
+  const topic_properties& properties) const {
+    // delete.retention.ms validation. Cannot be enabled alongside tiered
+    // storage.
+    if (!properties.delete_retention_ms.is_disabled()) {
```

Comment:
how does validation work when this has "default" and the value would then i guess be based on the cluster level configuration? or at this point does the topic property reflect that normalization?

Reply:
If `properties.delete_retention_ms` has been assigned using the cluster default (which we will assume for sake of argument here, does have an assigned value), then it would have been required that the cluster defaults for `cloud_storage_enable_remote_read` and `cloud_storage_enable_remote_read` were both `false`. So, validation would succeed in this case.

Example cases in which `topic_multi_property_validation()` would fail when a cluster default value for `delete_retention_ms` exists are explicitly invalid commands like:

* `rpk topic create t -c redpanda.remote.read=true`

* `rpk topic alter-config t --set redpanda.remote.read=true`

Reply:
thanks